src/main/java/creativei/entity/InquiryMarketing.java

```diff
@@ -15,6 +15,7 @@
     public InquiryMarketing(){}
 
     public InquiryMarketing(MarketingVo marketingVo){
+        this.setId(marketingVo.getId());
         this.marketingSource=MarketingSource.stringToEnum(marketingVo.getSource());
         this.isReferred=marketingVo.getReferred()==null?null:marketingVo.getReferred();
```

Comment:
avoid checking as we are setting null in case of null

Reply:
check is there so that if empty string is there in case of string field it should be set null

Reply:
issue resolved in commit 44f49a9